docs/kola-guide.md

```diff
@@ -0,0 +1,67 @@
+# Kola Introduction Documentation
+
+This document goes through the basics of how kola tests work and how someone just beginning can get started
+
+### What is Kola?
+
+Kola is a framework for testing software integration in CoreOS systems across multiple platforms. It is primarily designed to operate within the CoreOS Assembler for testing software that has landed in the OS image.
+
+> [Additional documentation about Kola](https://github.com/coreos/coreos-assembler/tree/main/docs/kola)
+
+### Steps to run a test from start to finish
+
+ 1. `cosa run` (Creates and brings you into the shell of a QEMU instance, running the provided version of FCOS or RHCOS)
+ 2. `cosa kola list` (Here we can see all of the tests that we can do)
+ 3. `cosa kola run <test-pattern>` (This will run a specific test and check to see if a test passes)
+ 4. `cosa kola run ext.config*` (This will run all the tests that start with `ext.config`)
+
+### More information on tests
+
+There are file logs in each of these locations: They will help you to debug the problem and will certainly give you hints along the way
+
+1. `journal.txt`
+2. `console.txt`
+3. `ignition.json`
+4. `gunzip journal-raw.txt.gz`
+
+### Extended artifacts
+
+1. Extended artefacts need additional forms of testing (You can pass the ignition and the path to the artefact you want to test)
```

Comment:
```suggestion
1. Extended artifacts need additional forms of testing (You can pass the ignition and the path to the artifact you want to test)
```

Reply:
Thanks for all of these suggestions Dusty!